docs/testingReaderRevenueComponents.md

```diff
@@ -0,0 +1,65 @@
+Testing Reader Revenue Components
+====================================
+
+## tl;dr
+
+- Check out these functions! https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/readerRevenueDevUtils.ts
+- Check out Bookmarklets in RRCP
+
+## Components
+
+www.theguardian.com is the main driver of traffic for our reader revenue sites, in particular support.theguardian.com where sign-ups for one-off and recurring contributions occur.
+
+The components which drive the most traffic are:
+
+### The epic
+- Sits in the document flow, at the bottom of article copy in articles and within the document flow of liveblogs
+- Typically only displays for a certain number of views in a given time window (normally 4 in 30 days) ***STILL RELEVANT?***
+- Won't display if the user is a signed in Recurring Contributor or is a Subscriber (important note: whether or not you are signed in has **no bearing** on the epic being shown if you have made a **one-off contribution**) ***STILL RELEVANT?***
```

Comment:
We could replace much of this with:
"Will not display if the user is signed in and has the `hideSupportMessaging` benefit, or the browser has the one-off contribution cookie from the last 90 days"